tests/integration/targets/setup_gnutar/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: uninstall gnu-tar
+  command: brew uninstall gnu-tar
```

Comment:
Shouldn't we use the homebrew module instead?

Reply:
Good idea, since we actually have one :) I copied this from ansible/ansible's CI, which cannot rely on that module (since it's in this collection).